ivy_tests/test_ivy/test_frontends/test_numpy/helpers.py

```diff
@@ -173,10 +173,9 @@ def test_frontend_function(*args, where=None, **kwargs):
 # noinspection PyShadowingNames
 def handle_where_and_array_bools(where, input_dtype, as_variable, native_array):
```

Comment:
I'd suggest we take in the `test_flags` object as a parameter instead of `as_variable` and `native_array`, this will make the code cleaner and more compact